src/main/java/com/endava/pages/SuccessStoriesPage.java

```diff
@@ -12,9 +12,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(SuccessStoriesPage.class);
 
-    public static String SUCCESS_STORY_LABEL = "SUCCESS STORY | RETAIL AND CONSUMER GOODS";
+    //Not sure which case is valid, so I'll keep both until we declare which one stay
```

Comment:
If the label has changed, please delete the old one

Reply:
Done.